components/gitpod-protocol/data/gitpod-schema.json

```diff
@@ -287,6 +287,10 @@
                                     "description": "Whether only stable, latest or both versions should be warmed up. Default is stable only."
                                 }
                             }
+                        },
+                        "vmoptions": {
+                            "type": "string",
+                            "description": "Configure VM options, for instance '-Xmx=4096m'."
```

Comment:
Could you also replace VM in .gitpod.yml to JVM 🙏 Just to avoid confusion that it is related to hardware, not JVM of IntelliJ

Reply:
Sure. Updated